docs/operator/config.md

```diff
@@ -24,16 +23,32 @@ export CORTEX_REGION="us-west-2"
 export CORTEX_NAMESPACE="cortex"
 
 # Image paths
+# <!-- CORTEX_VERSION_MINOR -->
```

Comment:
I think this will still show up in the view that the user sees since it is in the code block. You might have to move it above the code block